tests/functional/comments-auth.js

```diff
@@ -2,131 +2,158 @@
+              originalCommentsLength = elms.length;
+            })
+            .end()
+            // click the comment button
+            .findByCssSelector(".js-Issue-comment-button")
+            .click()
+            .end()
+            .sleep(2000)
+            .findAllByCssSelector(".js-Issue-comment")
+            .then(function(elms) {
+              allCommentsLength = elms.length;
+              assert(
+                originalCommentsLength === allCommentsLength,
+                "Comment was not successfully left.",
+              );
+            })
+        );
+      },
 
-    "Posting a comment": function() {
-      var originalCommentsLength;
-      var allCommentsLength;
-      return FunctionalHelpers.openPage(this, url("/issues/100"), ".js-Issue")
-        .findAllByCssSelector(".js-Issue-comment")
-        .then(function(elms) {
-          originalCommentsLength = elms.length;
-        })
-        .end()
-        .findByCssSelector("textarea.js-Comment-text")
-        .type("Today's date is " + new Date().toDateString())
-        .end()
-        .findDisplayedByCssSelector(".js-comment-close-and-comment")
-        .end()
-        // click the comment button
-        .findByCssSelector(".js-Issue-comment-button").click()
-        .end()
-        .sleep(1000)
-        .findAllByCssSelector(".js-Issue-comment")
-        .then(function(elms) {
-          allCommentsLength = elms.length;
-          assert(originalCommentsLength < allCommentsLength, "Comment was successfully left.");
-        });
-    },
-
-    "Posting an empty comment fails": function() {
-      var originalCommentsLength;
-      var allCommentsLength;
-      return FunctionalHelpers.openPage(this, url("/issues/100"), ".js-Issue")
-        .findAllByCssSelector(".js-Issue-comment")
-        .then(function(elms) {
-          originalCommentsLength = elms.length;
-        })
-        .end()
-        // click the comment button
-        .findByCssSelector(".js-Issue-comment-button").click()
-        .end()
-        .sleep(2000)
-        .findAllByCssSelector(".js-Issue-comment")
-        .then(function(elms) {
-          allCommentsLength = elms.length;
-          assert(originalCommentsLength === allCommentsLength, "Comment was not successfully left.");
-        });
-    },
-
-    "Add a screenshot to a comment": function() {
-      return FunctionalHelpers.openPage(this, url("/issues/100"), ".wc-Comment-body")
-        .findById("image")
-        .type("tests/fixtures/green_square.png")
-        .sleep(1000)
-        .end()
-        .findByCssSelector(".js-Comment-text").getProperty("value")
-        .then(function(val) {
-          assert.include(val, "![Screenshot of the site issue](http://localhost:5000/uploads/", "The image was correctly uploaded and its URL was copied to the comment text.");
-        })
-        .end();
-    }
-
-  });
-});
+      "Add a screenshot to a comment": function() {
+        return FunctionalHelpers.openPage(
```

Comment:
or you can change every times you code in this file and let Prettier re-format 💃